LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2018 Hideki Sekine
+
```

Comment:
Could you please delete the Copyright line like below PR?
https://github.com/rust-lang/rust/commit/2a8807e889a43c6b89eb6f2736907afa87ae592f#diff-7f0bdbc0a0545831259b66259ac6b604

The crate is derived from the rustc's `std` , so I'd like to follow their practice.

Reply:
I'll merge it first, then tweak as needed. 
Thank you.